tox.ini

```diff
@@ -0,0 +1,6 @@
+[flake8]
+#Black Compat
+max-line-length = 79
+extend-ignore = E203, W503
+#Django Compat
+exclude = .git,*migrations*
```

Comment:
I thought general PEP 8 guidelines dictated newlines at the end of a file. This isn't a python file, but maybe good to keep that as a standard. I have my editor set up to auto-add this anyways, so any file I touch will get one added.

Reply:
This is from: [Black Compatible Configs: flake8 ](https://black.readthedocs.io/en/stable/compatible_configs.html#id2)

> When breaking a line, Black will break it before a binary operator. This is compliant with PEP 8, but this behaviour will cause flake8 to raise W503 line break before binary operator warnings.
In some cases, as determined by PEP 8, Black will enforce an equal amount of whitespace around slice operators. Due to this, Flake8 will raise E203 whitespace before ':' warnings.
Since both of these warnings are not PEP 8 compliant, Flake8 should be configured to ignore these warnings via extend-ignore = E203, W503.

Reply:
Oops, misread your comment. I agree, I think I messed it up during git munging.